src/pip/_internal/utils/glibc.py

```diff
@@ -14,6 +14,19 @@ def glibc_version_string():
     # type: () -> Optional[str]
     "Returns glibc version string, or None if not using glibc."
 
+    try:  # https://github.com/pypa/pip/issues/6675#issue-463147612
+        glibc_version = os.confstr("CS_GNU_LIBC_VERSION").split()
```

Comment:
nitpick: `glibc_version` might be a misnomer, since it will be `("glibc", "2.17")`, for example.